packages/css-framework/src/components/_phase-banner.scss

```diff
@@ -0,0 +1,17 @@
+.rn-phase-banner {
+  background-color: color(primary, 100);
+  padding: spacing(2) spacing(6);
+  
+  &__text {
```

Comment:
I personally wouldn't nest selectors like this - my main reason is searching for selectors in an IDE.

I like to write out the full selector where possible (different for pseudo selectors).

Have we agreed some best practices around writing CSS and are they documented somewhere? Are we linting SASS?

Reply:
I agree. When using BEM it's generally a good idea to spell out the whole selector and I'd even go as far as not nesting  (see my Button PR). BEM does away with the need to nest and that then makes it easier to search for a selector in your IDE. You often realise you don't need the whole selector all and maybe just need `.thing__text` instead of `.thing.thing__text` which makes it easier for the browser to figure out the rule to use. We are linting sass but not looked at the config in a while

Reply:
I'm totally fine with that but I'd like to get @hxltrhuxze 's opinion on this matter before I change it.

Reply:
Totally fine with this naming convention. Whilst I like scoping stuff to use the `&__class` helper, I can see the advantages of spelling out the whole class name each time.

Reply:
Works for me.